contracts/src/v0.8/functions/dev/1_0_0/FunctionsSubscriptions.sol

```diff
@@ -184,9 +185,6 @@ abstract contract FunctionsSubscriptions is IFunctionsSubscriptions, IERC677Rece
     if (currentBalance < amount) {
       revert InsufficientBalance(currentBalance);
     }
-    if (s_totalLinkBalance < amount) {
```

Comment:
What is the reason these are removed?

Reply:
To save on save - in my opinion they don't really provide much. Alerting can't use it because it is not an event.

Reply:
This is concerning me slightly.  While I myself cannot conceive of a case where it would be triggered, I don't like that we are removing a check like this after audit.  Can we keep the check & revert, but remove the specific `TotalBalanceInvariantViolated` error to still save some size?

Reply:
Getting this back in will require slimming ~300 bytes. Just spent some time looking for size reductions and couldn't find any low hanging fruit. I'm open to suggestions of things we don't need